e2e-tests/insurance/cypress/e2e/journeys/policy-and-exports/change-your-answers/change-your-answers-different-name-fields.spec.js

```diff
@@ -0,0 +1,157 @@
+import { summaryList, field, submitButton } from '../../../../../../pages/shared';
+import partials from '../../../../../../partials';
+import { FIELD_IDS, FIELD_VALUES, ROUTES } from '../../../../../../constants';
+import { INSURANCE_ROOT } from '../../../../../../constants/routes/insurance';
+
+const {
+  POLICY_AND_EXPORTS: {
+    CHECK_YOUR_ANSWERS,
+    NAME_ON_POLICY_CHANGE,
+    DIFFERENT_NAME_ON_POLICY_CHANGE,
+  },
+} = ROUTES.INSURANCE;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      NAME_ON_POLICY: { NAME, POSITION },
+    },
+    ACCOUNT: { FIRST_NAME, LAST_NAME, EMAIL },
+  },
+} = FIELD_IDS;
+
```

Comment:
Can we make sure we have `INSURANCE_FIELD_IDS` and `INSURANCE_ROUTES` in all new E2E test?